static/sass/config/_scale.scss

```diff
@@ -1,33 +1,62 @@
 // Scale Settings
 // ==============
 
-// Dependencies
-// ------------
 
-@import '~accoutrement-scale/sass/scale';
-
-$ratios: (
+// Ratios
+// ------
+/// @group scale
+/// @preview ratios
+/// @link http://oddbird.net/accoutrement-scale/sassdoc/
+///   Accoutrement Scale
+$modular-scale: (
   'multiple': 'linear',
+  'text-ratio': 1.35,
 );
 
+@include herman-add-ratios('modular-scale', $modular-scale);
 
-$component-sizes: (
-  'full-content': 40rem,
-);
 
+// Text Sizes
+// ----------
+/// @group scale
+/// @preview sizes;
+/// @link http://oddbird.net/accoutrement-scale/sassdoc/
+///   Accoutrement Scale
+$text-sizes: (
+  'root': 18px,
+  'small': 'root' ('text-ratio': -1),
```

Comment:
"text-ratio" is established earlier as `1.35` — at this point somewhat arbitrary, but a ratio that looked good to me as a starting-point for line-height and modular scales.

So the "small" text-size is based on the "root" size (`18px`), but one position lower on the "text-ratio" scale — about `74%` or `13.3px`.